README.md

```diff
@@ -317,3 +318,38 @@ plus any dependencies referenced in the example itself are made relative to the
 * `exampleMap` - a hash map holding each example by id, which is a unique id generated from the name
 of the example
 
+
+## `typescript` Package
+
+### File Readers:
+
+* at the moment we are not using a filereader but the `readTypeScriptModules` processor to read our modules. 
+
+### Processors
+
+* `readTypeScriptModules` - parse the `sourceFiles` with the help of the `tsParser` service and return doc 
+for each exported member. Members marked with `/** @internal */` are not exported as well as members starting with
+an underscore (`_`)
+
+### Tag Definitions
+
+The `tsdoc` package will generate docs for the following types: 
+`class`, `interface`, `function`, `var`, `const`, `let`, `enum` and `type-alias`
```

Comment:
Tag Definitions are supposed to be definitions for the jsdoc style tags, such as `@internal`, rather than what "types" of docs are generated.